src/pymmcore_widgets/_pixel_configuration_widget.py

```diff
@@ -473,7 +473,9 @@ def __init__(self, parent: QWidget | None = None):
 
         self.horizontalHeader().setSectionResizeMode(QHeaderView.ResizeMode.Stretch)
         self.horizontalHeader().setVisible(False)
-        self.verticalHeader().setSectionResizeMode(QHeaderView.ResizeMode.Stretch)
+        self.verticalHeader().setSectionResizeMode(
+            QHeaderView.ResizeMode.ResizeToContents
+        )
```

Comment:
what happens if you use 

```suggestion
        self.verticalHeader().setDefaultSectionSize(20)
```

here instead?

I know it feels like a magic number (and it is; albeit a very common qt line height) ... however with resizeToContents... I see this (extra room at the bottom)

<img width="278" alt="Screenshot 2024-07-17 at 2 33 34 PM" src="https://github.com/user-attachments/assets/462fbf82-5ad3-4c69-9c5c-2b20318df6f5">

and with an explicit section size I see:

<img width="290" alt="Screenshot 2024-07-17 at 2 33 21 PM" src="https://github.com/user-attachments/assets/80444060-57bf-4f25-b7cb-63fe46b8a3fc">

what does explicit section size look like for you?

Reply:
It looks fine to me - switched it out in 802fe53. It must have been the `ResizeMode.Stretch` that was causing the problem.